pandas/io/sql.py

```diff
@@ -617,9 +618,11 @@ def _engine_builder(con):
 
         except ImportError:
             _SQLALCHEMY_INSTALLED = False
```

Comment:
Ah, no, sorry, this is good (the `_is_sqlalchemy_connectable` does more than just setting the global value to True, so that should still be run)